### PR TITLE
Created all models

### PR DESCRIPTION
**Pull request by Deenknight**
Event should be fully defined for what matters for halfway point.

### CODE REVIEW

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java
Comment (davidurschel) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:1]: Update the existing constructor to allow for initialization of this variable. Add a list to contain notifications, also, add a model for individual notifications. Update this model to make it easier to store in the document database, likely we can use the androidID as the document name for this implementation.

Comment (Deenknight) [reply napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:1]: The things that matter have been updated. The 2 constructors should be good now for Users and Events. Database handling is in the controller, not model for us.

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 1]
```diff
@@ -34,11 +84,33 @@ public void setName(String name) {
...
-        this.date = date;
+    public Date getLotteryDate() {
+        return lotteryDate;
     }
+
+    public String getDescription(){
+        return  description;
+    }
+
+    public int getEntrantLimit() {
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:99]: setter needed

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 2]
```diff
@@ -34,11 +84,33 @@ public void setName(String name) {
...
+        return eventDate;
     }
 
-    public void setDate(Date date) {
-        this.date = date;
+    public Date getLotteryDate() {
+        return lotteryDate;
     }
+
+    public String getDescription(){
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:95]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 3]
```diff
@@ -34,11 +84,33 @@ public void setName(String name) {
...
+        return  description;
+    }
+
+    public int getEntrantLimit() {
+        return entrantLimit;
+    }
+
+    public int getParticipantLimit() {
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:103]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 4]
```diff
@@ -34,11 +84,33 @@ public void setName(String name) {
...
 
-    public Date getDate() {
-        return date;
+    public Date getEventDate() {
+        return eventDate;
     }
 
-    public void setDate(Date date) {
-        this.date = date;
+    public Date getLotteryDate() {
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:91]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 5]
```diff
@@ -34,11 +84,33 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public Date getDate() {
-        return date;
+    public Date getEventDate() {
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:87]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 6]
```diff
@@ -26,6 +72,10 @@ public void setId(String id) {
         this.id = id;
     }
 
+    public String getOrganizerId(){
+        return organizerId;
+    }
+
     public String getName() {
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:79]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java [hunk 7]
```diff
@@ -26,6 +72,10 @@ public void setId(String id) {
         this.id = id;
     }
 
+    public String getOrganizerId(){
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/Event.java:75]: setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java [hunk 8]
```diff
@@ -10,6 +13,13 @@ public class User {
     private Boolean isAdmin;
     private Boolean isOrganizer;
 
+    private List<Notification> notifications;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:16]: needs getter setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java [hunk 9]
```diff
@@ -10,6 +13,13 @@ public class User {
...
+
+    private List<String> waitlist;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:18]: getter setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java [hunk 10]
```diff
@@ -10,6 +13,13 @@ public class User {
...
+    private List<String> chosen;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:19]: getter setter

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java [hunk 11]
```diff
@@ -10,6 +13,13 @@ public class User {
...
+    private List<String> cancelled;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:20]: does it need list of cancelled?

File: napkinapp/app/src/main/java/com/example/napkinapp/models/User.java [hunk 12]
```diff
@@ -10,6 +13,13 @@ public class User {
...
+    private List<String> registered;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/models/User.java:21]: getter setter

**Review by Sherali5543 — APPROVED** [review]

**Pull request merged**